### ISSUE TITLE
"Killed" after trying to create a new credential file.

### ISSUE THREAD

**Issue by evanh1122** [Hari-Nagarajan/fairgame#389]
**Describe the bug**
I tried installing the bot on Raspbian today, and when running it for the first time, it created a new credential file. However, after adding a password to the credential file, it says "Killed" and then stops the operation. I've never tried older versions on Raspbian so I'm unsure if this is a bug or my own issue.

**To Reproduce**
1. replace old files with these files: https://github.com/Hari-Nagarajan/fairgame/archive/0.4.zip
2. pipenv install
3. python app.py amazon --test
4. Enter in email and password for amazon
5. Enter in password for credential file
6. Sees error

**Expected behavior**
I've tried this new update on Windows and it works perfectly fine. After entering the password for the new credential file,  it says "Credentials safely stored," and continues to amazon.com and everything works perfectly fine.

**Screenshots**
https://imgur.com/a/gWUKtlm

**Desktop (please complete the following information):**
 - OS: Raspbian 32-bit
 - Browser chromium

**Additional context**
I've followed all the steps on listed on github for Raspberry Pi as well as this document: https://docs.google.com/document/d/1VUxXhATZ8sZOJxdh3AIY6OGqwLRmrAcPikKZAwphIE8/edit

**Comment by JustinShetty** [comment]
I'm seeing the same behavior on a rpi3b+ Using python3.8 as required. Seems like a platform-specific issue with the scrypt

**Comment by worst-mod-confirmed** [comment]
There are a ton of issues with rpi, and a few people working on fixing them. In theory, it should just work, but it doesn't. I would advise not using them for now, or set it up somewhere else while you troubleshoot.

**Comment by TaiPhamD** [comment]
if you issue sudo dmesg

```
[10532.145334] Out of memory: Killed process 25209 (python3) total-vm:1091068kB, anon-rss:506912kB, file-rss:16kB, shmem-rss:0kB, UID:1000 pgtables:526kB oom_score_adj:0
```

looks like there isn't enough memory to spawn this process. anyone more familiar with raspberry pi can chime in on how to get past this memory limitation?

**Comment by SalvadorSTM** [comment]
@TaiPhamD 
This worked for me on a Pi 3B+. This probably means the pi 4 with more RAM out of box should not have this error.

**Comment by TaiPhamD** [comment]
> @TaiPhamD
> This worked for me on a Pi 3B+. This probably means the pi 4 with more RAM out of box should not have this error.

Yup! I ordered a 8gb pi 4 to play with. Will be a nice upgrade from my 1gb pi 3

**Comment by remixmtg** [comment]
I was running into the same issue.  Increased the swapfile size and now instead of the "killed" message it just stays hung up and freezes. Followed the steps that OP did.

![Screen Shot 2021-01-20 at 11 30 32 PM](https://user-images.githubusercontent.com/77762350/105280397-aa388a00-5b77-11eb-8e2b-9ba9166ea9ee.png)
